positron/components/ModuleLoader.jsm

```diff
@@ -200,21 +199,29 @@ function ModuleLoader(processType, window) {
     }
   };
 
-  // The `process` global is complicated.  It's implemented as a module
-  // that uses `require` to import other modules, but it also needs to exist
-  // before those modules are evaluated, since it's a global that's supposed
-  // to be available in them.
+  // The `process` global is complicated.  It's implemented by a module,
+  // process.js, that we require.  But it's also supposed to exist before any
+  // modules are required, since it's a global that's supposed to be available
+  // in all of them.
```

Comment:
it's not entirely clear what "them" means here. Maybe change to "in all modules".